### DISCUSSION TITLE
Use of Symlinks with AO

### DISCUSSION THREAD

**Discussion by TinkerNZ** [kubilus1/autoortho#162] [Q&A]
Hi, because of space constraints on my C:/ drive where XPlane12 and AO are installed, I have my Custom Scenery files located on another drive (SSD) and use Symlinks back to the XPlane12/Custom Scenery.
When I set the Dir. for X-Plane scenery on the Setup page to XPlane 12/Custom scenery, which now has Symlinks only ( having moved the installed base files and Symlinked back) the Scenery page still shows the files as installed.
However when running AO it crashes XPlane12 as soon as I try and start XP. Doesn't even reach the menu screen for XP.
I now have the X-Plane scenery dir. set to the other SSD containing the actual files and when I click RUN I get a warning that the Custom Scenery location doesn't look correct and AO may not function correctly.
However, everything seems to work just fine and the Ortho scenery in Sim looks awesome (thanks for that). 
What might be wrong or should I just ignore the warning?

And as a side question, what is the best location in the scenery_packs.ini for the 'z_autoortho' folder? (it is not indicate in your sample order screenshot) Or is it irrelevant where it is in the order?

Cheers
TinkerNZ

**Comment by Jeff2122** [comment]
I get the warning too. Ignore that it thinks it doesn't "look right" when it starts. 
z_ortho is just below /yAutoOrtho_overlays/ and just above /z_eur_00/

**Reply by TinkerNZ** [reply]
Thanks for the prompt reply. Yes, I will continue to ignore the warning and make the slight ini order adjustment (just in case).
Very impressed with what AO is delivering and it can only get better.

Cheers
